charts/kong/example-values/minimal-k4k8s-with-kong-enterprise.yaml

```diff
@@ -0,0 +1,54 @@
+# Basic values.yaml for Kong for Kubernetes with Kong Enterprise
+# Several settings (search for the string "you-must") require user-provided
+# Secrets. These Secrets must be created before installation.
+#
+# This installation does not create an Ingress or LoadBalancer Service for
+# the Admin API. It requires port-forwards to access without further
+# configuration to add them, e.g.:
+# kubectl port-forward deploy/your-deployment-kong 8001:8001
+
+image:
+  repository: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
+  tag: 1.5.0.2-alpine
+  pullSecrets:
+    # you-must add this: https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-docker-registry-access
+    - kong-enterprise-edition-docker
+
+admin:
+  enabled: true
+  http:
+    enabled: true
+    servicePort: 8001
+    containerPort: 8001
+
+enterprise:
+  enabled: true
+  license_secret: you-must-create-a-license-secret
+  vitals:
+    enabled: false
+  portal:
+    enabled: false
+  rbac:
+    enabled: false
+  smtp:
+    enabled: false
+
+env:
+  prefix: /kong_prefix/
+  database: postgres
+  password:
+    valueFrom:
+      secretKeyRef:
+        name: you-must-create-an-rbac-bootstrap-secret
+        key: you-must-replace-with-your-key
```

Comment:
ditto for `name` and `key` here.

Reply:
Assume that most folks will not even look inside this values.yaml file.

Reply:
You have to for the Enterprise examples: all of them require at least one setting that has to change. I can align the ones that exist, but the whole purpose of these is to provide a values.yaml for users to look at that's targeted at one particular deployment pattern, since the standard values.yaml covers settings for all options at once.

Reply:
> all of them require at least one setting that has to change.

DB-less ones don't. We want to minimize the friction of standing up Enterprise as much as possible. 
What do you think about using the default values as the one listed in this guide:
https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/deployment/kong-enterprise.md#kong-enterprise-bootstrap-password